content/blog/2020-04-16-april-20-community-gems.md

```diff
@@ -0,0 +1,151 @@
+---
+title: April '20 Community Gems
+date: 2020-04-16
+description: |
+  A roundup of technical Q&A's from the DVC community. This month, we discuss 
+  the DVC cache, pipelines, cloud storage options and concurrency.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community. This month, we discuss 
+  the DVC cache, cloud storage options and concurrency.
+picture: ../../static/uploads/images/2020-04-16/DVC_Gems_April_20.png
```

Comment:
please, merge the latest master - I think the way we do URL has changed (simplified)

Reply:
okay just did that

Reply:
take a look at some other blog posts, please - it's a minor simplification update

Reply:
oh, you mean with the picture URL. got it! changing now

Reply:
picture and author probably

Reply:
also updated author. although that seems to have thrown a "restyled" error in CI

Reply:
not that, restyled fixed an extra space in one of the lines that was introduced